app/views/external_activities/matedit.html.haml

```diff
@@ -6,8 +6,18 @@
         %span.component_title
         #{@external_activity.name} Editing
     .action_menu_header_right
-      = link_to "Assign To Class", "javascript:void(0)",:onclick=>"get_Assign_To_Class_Popup(#{@external_activity.id}, '#{@external_activity.class.to_s}', null, true)", :class=>"button"
-      = link_to "Edit Resource Options", "javascript:void(0)",:onclick=>"get_edit_resource_popup(#{@external_activity.id}, {use_short_form: true, content_height: 400})", :class=>"button"
+      = link_to t("matedit.archive"),
+        archive_external_activity_path(@external_activity),
+        confirm: t("matedit.archive_confirm"),
+        :class=>"button"
```

Comment:
They liked it.  The didn't love the confirmation box, so I am working on a fix for that.